symfony/src/Service/HelpService.php

```diff
@@ -189,4 +189,9 @@ public function getDocumentInfo($id)
         }
         return false;
     }
+
+    public static function getFeedbackUrl(): string
```

Comment:
I could have set this as a class constant (e.g. how we handle `$confluenceResources`), but I figured there _might_ be a need in the future to have a dev/test version of this survey, and this would be an easier spot to add that without having to touch the other files.